webapp/client/src/lib/propTypes.js

```diff
@@ -1,4 +1,12 @@
 import PropTypes from "prop-types";
+import FieldLabelScaffolding from "../components/FieldLabelScaffolding";
+
+// Used if the field is handed to a more abstract component.
+// e.g. MerkzeichenPersonAPage -> MerkzeichenPage
+const abstractedFieldPropType = PropTypes.exact({
+  name: PropTypes.string.isRequired,
+  label: FieldLabelScaffolding.propTypes.label,
+});
```

Comment:
Yes, I wanted to talk to you about this warning anyway, because I could not find another way to get rid of the warning, too. So, I would be very happy to change that for this and all other extendedFieldPropTypes